input/fsh/omrs-diagnosticReport.fsh

```diff
@@ -3,6 +3,58 @@ Parent: DiagnosticReport
 Id: OpenMRS-diagnostic-report
 Title: "OpenMRS Diagnostic Report"
 Description: "DiagnosticReport resource created by a OpenMRS that representing results of diagnostic tests that are performed in a given clinical context."
+* identifier 0..*
+* identifier ^slicing.discriminator.type = #value
+* identifier ^slicing.discriminator.path = "system"
+* identifier ^slicing.rules = #open
+* identifier only OMRSPatientIdentifier
+* basedOn 0..*
 * code.coding 1..*
 * code.coding from OpenMRSOrderCodeVS
 * result only Reference(OpenMRSObservation)
+* status 1..1
+* category 0..*
+* encounter 0..1
+* subject 0..1
+* issued  0..1
+* result  0..*
+* effectiveDateTime 0..1
+* performer 0..*
+* resultsInterpreter 0..*
+* specimen 0..*
+* result 0..*
+* imagingStudy 0..*
+* media 0..*
+* conclusion 0..1
+* presentedForm 0..*
+
+Instance: example-Openmrs-DiagnosticReport
+InstanceOf: OpenMRSDiagnosticReport
+Usage: #example
+Title: "Openmrs DiagnosticReport"
+Description: "Example OMRS DiagnosticReport Resource"
+* identifier.id = "53fbd0ef-ae10-46a0-9d0a-387917a66d6e"
```

Comment:
Does the FHIR2 module translate into a diagnostic report with an identifier? I don't see it anywhere in the translator: https://github.com/openmrs/openmrs-module-fhir2/blob/master/api/src/main/java/org/openmrs/module/fhir2/api/translators/impl/DiagnosticReportTranslatorImpl.java

Reply:
I also don't see any identifiers in the example DR from the fhir2 tests: https://github.com/openmrs/openmrs-module-fhir2/blob/master/test-data/src/test/resources/org/openmrs/module/fhir2/providers/DiagnosticReport_create_r4.json